src/settings.js

```diff
@@ -1414,6 +1414,10 @@ var WASM2JS = 0;
 // -fsanitize=undefined. To use minimal runtime, also pass `-fsanitize-minimal-runtime`.
 var UBSAN_RUNTIME = 0;
 
+// Whether we should load the WASM source map at runtime.
+// This is enabled automatically when using -g4 with sanitizers.
+var SOURCE_MAPS = 0;
```

Comment:
I think this doesn't need to be plural (so just `_MAP`).

The name still seems ambiguous, though. How about `LOAD_SOURCE_MAP`?